docs/platform/howto/pro-platform.rst

```diff
@@ -0,0 +1,7 @@
+Pro Platform
+=============
+
+Pro Platform gives enterprise organizations added capabilities for implementing security, resilience, reliability, and cost-efficiency strategies for their data infrastructure. Pro Platform features are available across disaster recovery, security and compliance, advanced networking, and user lifecycle management.
+
+To enable Pro Platform for your organization and use Pro features, contact the sales team at sales@Aiven.io.  
```

Comment:
should Aiven be capitalized in the email? thought it's just `bla@aiven.io`

Reply:
This is based on Value rules and is done elsewhere in the docs like this, yes.